feature-utils/poly-sparql/src/triple-generation.js

```diff
@@ -0,0 +1,66 @@
+import { v4 as uuid } from "uuid";
+
+export function jsObjectToTriplesString(
+    subject,
+    predicate,
+    jsObj,
+    blankNodeString = "_:" + uuid()
+) {
+    let triplesString = `${subject} ${predicate} ${blankNodeString} `;
+    for (let [key, value] of Object.entries(jsObj)) {
+        const keyPredicate = "poly:" + key;
+        if (typeof value === "object") {
+            if (Array.isArray(value)) {
+                triplesString += jsArrayToTriplesString(
+                    blankNodeString,
+                    keyPredicate,
+                    value
+                );
+                continue;
+            }
+            triplesString += jsObjectToTriplesString(
+                blankNodeString,
+                keyPredicate,
+                value
+            );
+            continue;
+        }
+        triplesString += `${blankNodeString} ${keyPredicate} ${
+            typeof value === "string" ? `"${value}"` : value
+        } `;
```

Comment:
This means that every string is going to have a whitespace at the end. I can live with that, but I would probably try to not assume that the previous string is whitespace-padded and take care of it every time something is concatenated.